README.md

```diff
@@ -27,21 +27,34 @@ environment.
     npm i
     ```
 
-3. Firing up the server diverge a little depending upon the OS you're using.
+3. After installing the depedencies, run
```

Comment:
Merge this step with `2`. You can ask the users to **only** run `npm run setup` and that will take care of everything. Also, add a note that users can run `npm i` as well if they need to update the dependencies.

Reply:
@algomaster99 
I have updated the documentation for this. Let me know if there are any changes required.